tests/FINDOLOGIC/Export/Tests/DummyNumericValue.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace FINDOLOGIC\Export\Tests;
+
+use FINDOLOGIC\Export\Helpers\UsergroupAwareNumericValue;
+
+class DummyNumericValue extends UsergroupAwareNumericValue
+{
+    /**
+     * @return string The name of the value , e.g. name or description, which is used for more accurate error
```

Comment:
Remove the additional space in `value , e.g.` and use only one line for the doc entry